lib/internal/streams/add-abort-signal.js

```diff
@@ -18,13 +18,13 @@ const validateAbortSignal = (signal, name) => {
   }
 };
 
-function isStream(obj) {
+function isReadable(obj) {
```

Comment:
unrelated change - just renamed the function, to make what it really checks clearer

Reply:
This is not correct, the function is named `isStream` to check if it is an instance of the `Stream` contract, which unfortunately has to have a `.pipe()` method.

Reply:
Oh, my bad - will revert. It's just exactly the same as `isReadable` in `utils`.